runtime/interpreter/interpreter.go

```diff
@@ -4534,7 +4534,7 @@ func (interpreter *Interpreter) mapMemberValueAuthorization(self Value, memberAc
 }
 
 func (interpreter *Interpreter) getMemberWithAuthMapping(self Value, locationRange LocationRange, identifier string) Value {
-	result := interpreter.getMember(self, locationRange, identifier)
+	result := interpreter.getMember(self, locationRange, identifier, nil)
```

Comment:
Should we pass `UnauthorizedAccess` here instead of `nil`?

Reply:
This argument isn't actually needed anymore (see comment below), so I just removed it.